gallery/FPCNCoactivations.py

```diff
@@ -50,20 +51,19 @@
 def init_frontend():
     nl = NeurolangPDL()
 
+
     nl.add_symbol(
         np.log,
         name="log",
         type_=Callable[[float], float],
     )
-    nl.add_symbol(
-        lambda it: float(sum(it)),
-        name="agg_sum",
-        type_=Callable[[Iterable], float],
-    )
 
     @nl.add_symbol
-    def agg_count(*iterables) -> int:
-        return len(next(iter(iterables)))
```

Comment:
The `sum` and `count` aggregation are now built-in aggregations. No need to add them again (see https://github.com/NeuroLang/NeuroLang/blob/master/neurolang/datalog/aggregation.py#L51)

Reply:
Good to know. Thanks